scripts/package.json

```diff
@@ -11,6 +11,9 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "fs": "0.0.1-security",
```

Comment:
Added npm-shringwrap.json and the package.json file in temp_modules that were created when running npmx generate.